gatsby-theme-troubleshoot/gatsby-config.js

```diff
@@ -54,7 +54,7 @@ module.exports = options => ({
         algoliaAppId: "NHTIKUBZ4C",
         baseUrl: "https://troubleshoot.sh",
         logoLink: "https://troubleshoot.sh",
-        gaTrackingId: "UA-61420213-14",
+        gaTrackingId: process.env.GA_TRACKING_ID,
```

Comment:
Does this gracefully handle when the env isn't present? I assume that would just be null, and everything would be fine, but just want to make sure.

Reply:
Yes, null values will just be ignored